Library/Formula/ghc.rb

```diff
@@ -4,6 +4,8 @@ class Ghc < Formula
   url "https://downloads.haskell.org/~ghc/7.10.2/ghc-7.10.2-src.tar.xz"
   sha256 "54cd73755b784d78e2f13d5eb161bfa38d3efee9e8a56f7eb6cd9f2d6e2615f5"
 
+  revision 1
+
   bottle do
     revision 1
```

Comment:
Formula revisions force the update on everyone. Bottle revisions are just an internal mechanism to refresh the bottle for whatever reason (Updated resources, non-urgent patching, configuration changes, sha1 => sha256, etc etc). Despite the wording they are essentially two different functions.